### PR TITLE
Change how group profile is loaded

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

File: orcid-core/src/main/java/org/orcid/core/manager/impl/ProfileEntityManagerImpl.java [hunk 1]
```diff
@@ -82,7 +84,8 @@
     private JpaJaxbWorkAdapter jpaJaxbWorkAdapter;
 
     @Override
-    public ProfileEntity findByOrcid(String orcid) {
+    @Cacheable(value = "profile-entity", key = "#orcid.concat('-').concat(#lastModified)")
```
Comment (amontenegro) [inline orcid-core/src/main/java/org/orcid/core/manager/impl/ProfileEntityManagerImpl.java]: Hey @rcpeters,

https://trello.com/c/T4VGtkzU/1702-change-the-way-groupprofile-is-loaded

Please check this, now, when we get the group info from the ClientDetailsEntity class, we just get an orcid string, then, we call this to get the ProfileEntity, which, hopefully will be already cached.

Thanks

**Comment by amontenegro** [comment]
https://trello.com/c/T4VGtkzU/1702-change-the-way-groupprofile-is-loaded

**Comment by rcpeters** [comment]
@amontenegro Sorry, but we shouldn't use the cacheable annotation any longer. See /orcid-core/src/main/java/org/orcid/core/manager/impl/OrcidProfileCacheManagerImpl.java

**Comment by amontenegro** [comment]
@rcpeters 

Done, now we have a ProfileEntityCacheManager that will cache profile entities, and, all classes that needs a profile entity use it instead of directly calling the ProfileEntityManager

Thanks

**Pull request merged**